### PR TITLE
Create Representative Body detail page

### PR DESCRIPTION
**Pull request by anitacaron**

### CODE REVIEW

**Review by Windvis — CHANGES_REQUESTED** [review]

File: app/routes/organizations/organization.js [hunk 1]
```diff
@@ -0,0 +1,7 @@
+import Route from '@ember/routing/route';
+
+export default class OrganizationsOrganizationRoute extends Route {
+  async model(params) {
+    return this.store.findRecord('representative-body', params.id);
```
[the remark this answers is not in the document]
Comment (anitacaron) [reply app/routes/organizations/organization.js:5]: It's a temporary thing. I was waiting for this second comment 😬  I'll change, sir 😄

File: app/templates/organizations/organization/index.hbs [hunk 2]
```diff
@@ -0,0 +1,127 @@
...
+                  <:content>
+                    <AuLink
+                      @route="administrative-units.administrative-unit"
+                      @model={{organization.id}}
+                    >
+                      {{organization.name}}
+                    </AuLink>
+                  </:content>
+                </Item>
+            {{/each}}
```
[the remark this answers is not in the document]
Comment (anitacaron) [reply app/templates/organizations/organization/index.hbs:120]: 🙏🏼

File: app/router.js [hunk 3]
```diff
@@ -76,6 +76,9 @@ Router.map(function () {
       this.route('new', { path: '/nieuwe-bestuurseenheid' });
     }
   );
+  this.route('organizations', { path: '/organisaties' }, function () {
```
[the remark this answers is not in the document]
Comment (anitacaron) [reply app/router.js:79]: Suggestions?

[the remark this answers is not in the document]
Comment (Windvis) [reply app/router.js:79]: Yea, not sure to be honest. I guess `/ondernemingen-and-verenigingen` would be overkill? :smile:

File: app/routes/organizations/organization.js (hunk 1 above)
[the remark this answers is not in the document]
Comment (Windvis) [reply app/routes/organizations/organization.js:5]: haha, well it's just my preference. You are allowed to disagree and not change it :smile:.

File: app/router.js (hunk 3 above)
[the remark this answers is not in the document]
Comment (Windvis) [reply app/router.js:79]: Maybe just keep it like this for now.

[the remark this answers is not in the document]
Comment (anitacaron) [reply app/router.js:79]: yeah, it's a temporary thing 😬

File: app/routes/organizations/organization.js (hunk 1 above)
[the remark this answers is not in the document]
Comment (anitacaron) [reply app/routes/organizations/organization.js:5]: No worries, you're right.

**Review by Windvis — APPROVED** [review]

**Pull request merged**